tools/update-eslint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# Shell script to update ESLint in the source tree to the latest release.
+
+# Depends on npm and node being in $PATH.
+
+# This script must be be in the tools directory when it runs because it uses
+# $BASH_SOURCE[0] to determine directories to work in.
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+rm -rf eslint
+
+# eslint-plugin-markdown is pinned at 1.0.0-beta.4 until there is a release
+# that fixes https://github.com/eslint/eslint-plugin-markdown/issues/69.
+npm install --production eslint@latest eslint-plugin-markdown@1.0.0-beta.4
+rm package-lock.json 
+
+# Install dmn if it is not in path.
+type -P dmn || npm install -g dmn
+
+# Use dmn to remove some unneeded files.
+dmn -f clean
+
+# This symlink will be pointed at the wrong place after all the moving after 
+# everything is moved around so let's remove it.
+rm node_modules/.bin/eslint
```

Comment:
Have we tried passing `--no-bin-links` to `npm install` to prevent the symlink being created in the first place?